src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsClient/DiagnosticsClient.cs

```diff
@@ -367,6 +367,36 @@ internal ProcessInfo GetProcessInfo()
             return GetProcessInfoFromResponse(response, nameof(GetProcessInfo));
         }
 
+        internal bool TryParseVersion(out Version version)
```

Comment:
nit: Rename this (maybe TryGetProcessClrVersion), since the method does more than just parsing.

Reply:
I think this makes more sense to be an extension method of the ProcessInfo class rather than adding it to DiagnosticsClient.